src/custom.ts

```diff
@@ -16,6 +16,10 @@ export function createError(msg: string): Fail {
   return createFail(failSymbol, msg)
 }
 
+export type Meta = Readonly<{ type: 'custom'; isPure: false }>
+
+const meta: Meta = { type: 'custom', isPure: false }
```

Comment:
Idea (to be implemented later): To support custom runtypes fully in schema creation, we only have to allow custom metadata on them and add a callback in our `toXxxSchema` functions for custom runtypes.